src/views/compute/DeployVM.vue

```diff
@@ -80,6 +80,29 @@
                         :options="keyboardSelectOptions"
                       ></a-select>
                     </a-form-item>
+                    <a-form-item
```

Comment:
@utchoang cc @DaanHoogland - the hypervisor would probably only be available once (a) the template is selected or (b) ISO is selected and hypervisor is selected; should this be moved after the template section then?

Reply:
@rhtyd I moved it below the template/iso section